preconditions/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.palantir.revapi'
 dependencies {
     api project(':safe-logging')
     compileOnly 'com.google.code.findbugs:jsr305'
+    implementation 'org.jetbrains:annotations'
```

Comment:
I'm not familiar enough with how to think about how annotations work in the context of the consumer of a library to be sure that `implementation` is correct here vs `api`. Why do the errorprone annotations need to be different? Is it explicitly because `CompileTimeConstant` annotates a parameter of the method signature rather than the method itself?

Reply:
I’m not entirely sure why errorprone annotations are api, I did verify that this functions correctly using wc internally